Sources/MongoSwiftSync/ClientSession.swift

```diff
@@ -96,4 +96,45 @@ public final class ClientSession {
     public func advanceOperationTime(to operationTime: Timestamp) {
         self.asyncSession.advanceOperationTime(to: operationTime)
     }
+
+    /**
+     * Starts a multi-document transaction for all subsequent operations in this session. Any options provided in
+     * `options` override the default transaction options for this session and any options inherited from
+     * `MongoClient`. The transaction must be completed with `commitTransaction` or `abortTransaction`. An in-progress
+     * transaction is automatically aborted when `ClientSession` goes out of scope.
+     *
+     * - Parameters:
+     *   - options: The options to use when starting this transaction
+     *
+     * - Throws:
+     *   - `CommandError` if an error occurs that prevents the command from executing.
+     *   - `LogicError` if the session already has an in-progress transaction.
+     *   - `LogicError` if `startTransaction` is called on an ended session.
+     */
+    public func startTransaction(_ options: TransactionOptions?) throws {
```

Comment:
let's provide a default value of `nil` for the options here.

Reply:
Ok, will do!

Reply:
I think this change along with the underscore removal got reverted by mistake.

Reply:
Thanks for catching that - will revert.